base/shared/annotation.js

```diff
@@ -313,10 +313,11 @@ var WidgetAnnotation = (function WidgetAnnotationClosure() {
     var dict = params.dict;
     var data = this.data;
 
-    data.fieldValue = stringToPDFString(
-      Util.getInheritableProperty(dict, 'V') || '');
+    var rawValue = Util.getInheritableProperty(dict, 'V') || '';
+    var value = (rawValue.name ? rawValue.name : rawValue) || '';
```

Comment:
This is only doing a truthy check. Is there a chance that rawValue.name is a falsey value like `0` or `""`? If there is, then there are cases where `rawValue = {name: ''}`  will make value equal to `{name: ''}`, when really it should have been `""`.

So I wonder if what you really want is:
`var value = (typeof(rawValue.name) === 'string' ? rawValue : rawValue.name) || '';`